frontend/packages/dev-console/src/components/EmptyState.tsx

```diff
@@ -83,7 +83,7 @@ const ODCEmptyState: React.FC<Props> = ({
       <div className="odc-empty-state__content">
         {allImportResourceAccess && (
           <CatalogTile
-            onClick={(e: Event) => navigateTo(e, '/import?importType=git')}
+            onClick={(e: any) => navigateTo(e, '/import?importType=git')}
```

Comment:
Avoid `any` if possible. Does `onClick` not have types in Patternfly?

Reply:
thanks @spadgett. this pr is currently stubs out the catalog tiles. its for propertiessidepanel, verticaltabs, and filtersidepanel. @rebeccaalpert is following up with the catalog tile components.

Reply:
@spadgett All components are now added to this pr, and the above type comment has been addressed.